lib/cucumber/formatter/ansicolor.rb

```diff
@@ -86,14 +86,25 @@ module ANSIColor
       # example:
       #
       #   apply_custom_colors('passed=white')
-      def apply_custom_colors(colors)
+      def self.apply_custom_colors(colors)
         colors.split(':').each do |pair|
           a = pair.split('=')
           ALIASES[a[0]] = a[1]
         end
       end
       apply_custom_colors(ENV['CUCUMBER_COLORS']) if ENV['CUCUMBER_COLORS']
 
+      # Provide an instance method of the same name, in case users are using it.
+
+      # Apply the custom color scheme
+      #
+      # example:
+      #
+      #   apply_custom_colors('passed=white')
+      def apply_custom_colors(colors)
+        ANSIColor.apply_custom_colors(colors)
```

Comment:
It's not strictly speaking good practice to invoke things direct on modules. Especially if they're not module functions.